addons/frontend/src/typst.css

```diff
@@ -38,6 +38,18 @@ body {
   background-color: #fff;
 }
 
+#typst-app.invert-colors {
+  filter: invert(0.95) hue-rotate(180deg);
+}
+
+#typst-app.invert-colors .typst-image {
+  filter: invert(0) hue-rotate(0deg);
+  transition: filter 0.1s ease-in-out;
+}
+
+#typst-app.invert-colors .typst-image:hover {
+  filter: invert(0.95) hue-rotate(180deg);

```

Comment:
will two `invert(0.95)` bring the image to its initial state?

Reply:
I guess no. invert(f) means `new_color = color * (1 - f) + 255 * f`. It's not invertable unless f=0 or 1

Reply:
f which is closer to 0.5 will make the color looks more "grey". Because when f=0.5 it's a average blend of 255 and color.

Reply:
That's shouldn't be a big problem though. both invert(1) and invert(0.95) is ok for me.

Reply:
> invert(f) means `new_color = color * (1 - f) + 255 * f`.

It is wrong, `invert` is a dual mix function, where `new_color =  color * (1 - f) + (255 - color) * f`.

> It's not invertable unless f=0 or 1

It is correct, but it is not a big problem in that fake dark mode already been making colors loss a bit.

Reply:
> That's shouldn't be a big problem though. both invert(1) and invert(0.95) are ok for me.

I considered but I was too lazy to calculate them. Perhaps we just make image's hovering filter to `invert(1)` to make it more perfect.